app/assets/javascripts/form_validations.js

```diff
@@ -42,8 +37,16 @@ var constraints = {
 document.addEventListener('turbolinks:load', function() {
   var forms = document.querySelectorAll('.form');
   if (!forms) return;
-
+  
   forms.forEach(function(form) {
+    
+    form.addEventListener("submit", function(ev) {
+      ev.preventDefault();
+      handleFormSubmit(form);
+      ev.stopPropagation();
```

Comment:
Yeah I'm not even sure why it's needed but if I don't stop the bubbling then after clicking submit on an incorrect form you can no longer submit even after fixing the errors. I can't figure out why so until I can this is a dirty fix.